ldap/servers/slapd/result.c

```diff
@@ -1999,6 +1999,64 @@ notes2str(unsigned int notes, char *buf, size_t buflen)
     return (buf);
 }
 
+static void
+log_op_stat(Slapi_PBlock *pb)
+{
+
+    Connection *conn = NULL;
+    Operation *op = NULL;
+    Op_stat *op_stat;
+    struct timespec duration;
+    char stat_etime[ETIME_BUFSIZ] = {0};
+
+    slapi_pblock_get(pb, SLAPI_CONNECTION, &conn);
```

Comment:
IMHO We should just return here if config_get_statlog_level is 0

Reply:
absolutely